manim/config.py

```diff
@@ -239,3 +239,47 @@ def get_camera_configuration(args):
         camera_config["background_opacity"] = 0
 
     return camera_config
+
+
+def initialize_directories(cfgargs):
+    dir_config = {}
+    dir_config["media_dir"] = cfgargs["media_dir"] or dirs.MEDIA_DIR
+    dir_config["video_dir"] = cfgargs["video_dir"] or dirs.VIDEO_DIR
+    print(not(cfgargs["video_dir"] or dirs.VIDEO_DIR))
+
+    if not (cfgargs["video_dir"] and cfgargs["tex_dir"]):
```

Comment:
We should discuss this more, because it's still really confusing.

Reply:
What exactly? The usage of or in the assignment?

Reply:
Work on this will be resumed later in a separate issue and PR about folder structures.